build.sh

```diff
@@ -62,6 +62,21 @@ check_prereqs()
 
 build_coreclr()
 {
+    #determine nproc in a platform independent way
+    NPROCS=1
+    OS=`uname`
+
+    if [ $OS = "Linux" ]; then
+      NPROCS=`grep -c ^processor /proc/cpuinfo`
```

Comment:
I think this has been already address in a slightly different way in https://github.com/dotnet/coreclr/pull/100